src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.security.InvalidAlgorithmParameterException;
+import java.util.MissingFormatArgumentException;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke \nWhat can I do for you?");
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        String userInput;
+        while(in.hasNext()){
+            userInput = in.nextLine();
+            if(userInput.equals("bye")){
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            } else if(userInput.contains("list")) {
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.print(i + 1 + ".");
+                    System.out.println(tasks.get(i).toString());
+                }
+            } else if (userInput.contains("todo")) {
+                String info = userInput.substring(5).trim();
+                tasks.add(new Todo(info));
+                int finalTask = tasks.size() - 1;
+                System.out.println("Got it. I've added this task:");
+                System.out.println(tasks.get(finalTask).toString());
+                if (tasks.size() == 1) {
+                    System.out.println("Now you have " +tasks.size()+" task in the list.");
+                } else {
+                    System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+                }
+            } else if (userInput.contains("deadline")) {
+                String [] listArray = userInput.split("/",2);
+                String description = listArray[0];
+                String dueDate = listArray[1];
+                String info = description.substring(8).trim();
+                String due = dueDate.substring(3).trim();
+                tasks.add(new Deadline(info, due));
+                int finalTask = tasks.size() - 1;
+                System.out.println("Got it. I've added this task:");
+                System.out.println(tasks.get(finalTask).toString());
+                if (tasks.size() == 1) {
+                    System.out.println("Now you have " + tasks.size() + " task in the list.");
+                } else {
+                    System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+                }
+            } else if (userInput.contains("event")) {
+                String [] listArray = userInput.split("/",3);
+                String description = listArray[0];
+                String startTime = listArray[1];
+                String endTime = listArray[2];
```

Comment:
User might not give the desired input. Consider implement some error handling taught in week 5